pages/enforcement/update/schema/index.js

```diff
@@ -18,8 +18,10 @@ const getSchema = subject => {
     });
   }
 
-  if (subject.profile.projects.length) {
-    subject.profile.projects.forEach(project => {
+  const flaggableProjects = (subject.profile.projects || []).filter(p => p.status !== 'inactive');
+
+  if (flaggableProjects.length) {
```

Comment:
There's no particular need for the `if` here. Running a `forEach` on an empty array is a no-op anyway, and it's guaranteed to be an array of some kind.